master/internal/api_experiment.go

```diff
@@ -114,7 +114,9 @@ func (a *apiServer) GetExperiment(
 		// easy deducible how long that would block. So the best we can really do is return without
 		// an error if we're in this case and log. This is a debug log because of how often the
 		// happens when polling for an experiment to end.
-		if !strings.Contains(err.Error(), actorDidNotRespond) {
+		if !strings.Contains(err.Error(), actorDidNotRespond) ||
+			!strings.Contains(err.Error(), "could not be found") {
+			// REMOVEME: the actor could not be found issue seems to be allowing exp delete test to pass
```

Comment:
It's not impossible that this could be the test legitimately seeing a race condition. E.G.
* we query DB state, it is active
* we get de-scheduled by the go run time
* experiment finishes it inbox, runs poststop https://github.com/determined-ai/determined/blob/master/master/pkg/actor/ref.go#L330-L337
* and sends it parent that it stopped https://github.com/determined-ai/determined/blob/master/master/pkg/actor/ref.go#L344
* then the parent ('experiments' actor) wakes up and deletes the experiment from the system https://github.com/determined-ai/determined/blob/master/master/pkg/actor/ref.go#L245
* then we wake back up and get "could not be found" rather than "did not respond" (which happens if the experiment gets the message but it was already closing).

so basically.. keep this.